File: 프로그래머스/0/120825. 문자 반복 출력하기/문자 반복 출력하기.cs
using System;

public class Solution 
{
    public string solution(string my_string, int n) 
    {
        string result = "";
        for(int i=0; i<my_string.Length; i++)
        {
            for(int j=0; j<n; j++)
            {
                result += my_string[i].ToString();
            }
        }
        return result;
    }
}

File: 프로그래머스/0/120825. 문자 반복 출력하기/문자 반복 출력하기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string, int n) 
{
    string result = "";
    for(char &c : my_string)
    {
        for(int i=0; i<n; i++)
        {
            result += c;
        }
    }
    return result;
}